app/javascript/stylesheets/home.css

```diff
@@ -61,25 +61,22 @@
 }
 
 .podcast {
+  background: url('../../assets/images/waveform.svg');
+  background-repeat: repeat-x;
+  background-position: center;
+  background-size: auto 85%;
+
   @apply bg-wnbrb-pink
+    py-20
     my-28
+    flex
+    items-center
+    justify-center
```

Comment:
These styles position the podcast tile in the center of that div, with padding on the top and bottom.